README.md

```diff
@@ -1 +1,31 @@
-# Proj_i02
\ No newline at end of file
+# Proj_i02
+
+
+
+
+
+## Installation Instructions
+
+1. Install [python] 2.7.10 or above, [git], [virtualenv], in your computer, if you don't have it already.
```

Comment:
Migrate this project setup to python 3, since development support of python 2 is over.